src/main/java/org/sagebionetworks/web/server/servlet/AppConfigServlet.java

```diff
@@ -0,0 +1,155 @@
+package org.sagebionetworks.web.server.servlet;
+
+import com.amazonaws.services.appconfigdata.AWSAppConfigData;
+import com.amazonaws.services.appconfigdata.model.GetLatestConfigurationRequest;
+import com.amazonaws.services.appconfigdata.model.GetLatestConfigurationResult;
+import com.amazonaws.services.appconfigdata.model.StartConfigurationSessionRequest;
+import com.amazonaws.services.appconfigdata.model.StartConfigurationSessionResult;
+import com.google.gwt.thirdparty.guava.common.base.Supplier;
+import com.google.gwt.thirdparty.guava.common.base.Suppliers;
+import com.google.inject.Inject;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.sagebionetworks.StackConfiguration;
+import org.sagebionetworks.aws.AwsClientFactory;
+
+public class AppConfigServlet extends HttpServlet {
+
+  public AWSAppConfigData appConfigDataClient;
+  public Supplier<String> configSupplier;
+  public String configurationToken;
+  private StackConfiguration stackConfiguration;
+  private final String defaultConfigValue =
+    "{\"default configuration\":\"true\"}";
```

Comment:
rename to reflect that it's a constant. What do you think about making the value an empty JSON object, since the actual config should always be non-empty?
```suggestion
  private final String DEFAULT_CONFIG_VALUE = "{}";
```